vapi/appliance/logging/forwarding.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0.
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logging
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/vmware/govmomi/vapi/rest"
+)
+
+const ApplianceLoggingForwardingPath = "/appliance/logging/forwarding"
+
+// Forwarding provides convenience methods to configure appliance logging forwarding.
+type Forwarding struct {
+	*rest.Client
+}
+
+// NewForwarding creates a new Forwarding with the given client
+func NewForwarding(client *rest.Client) *Forwarding {
```

Comment:
While there's only a few govmomi/vapi packages, we've been doing a package level `Manager` type + `func NewManager`. I think you could do that here too?